assets/queries/cloudFormation/s3_bucket_logging_disabled/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": "S3_bucket_logging_disabled",
+  "queryName": "S3 Bucket Logging Disabled",
+  "severity": "MEDIUM",
+  "category": "Logging",
+  "descriptionText": "S3 Bucket should have server access logging enabled",
```

Comment:
In the description, try something like "Server Access Logging must be enabled on S3 Buckets so that all changes are logged and trackable"